config_src/drivers/nuopc_cap/mom_ocean_model_nuopc.F90

```diff
@@ -610,6 +610,11 @@ subroutine update_ocean_model(Ice_ocean_boundary, OS, Ocean_sfc, &
     call finish_MOM_initialization(OS%Time, OS%dirs, OS%MOM_CSp, OS%restart_CSp)
   endif
 
+  if (do_thermo) &
+    call call_tracer_set_forcing(OS%sfc_state, OS%fluxes, OS%Time, &
```

Comment:
Is this call needed when USE_CFC_CAP=False? Does it change answers for cases without CFCs?

Reply:
It doesn't change answers for cases without CFCs, because `call_tracer_set_forcing` at this point only calls `CFC_cap_set_forcing`, and only if `USE_CFC_CAP=True`.
But I think it makes sense to 'always' `call_tracer_set_forcing`, so hooks to other tracers modules are added to it, like for MARBL tracers, the call to `call_tracer_set_forcing` doesn't get complicated.
I put always in quotes because of the `do_thermo` check. I put it inside the `do_thermo` check to be consistent with how we were previously calling `CFC_cap_fluxes`. The call to `CFC_cap_fluxes` was inside `convert_IOB_to_fluxes` which is only called if `do_thermo=.true.`.